README.md

```diff
@@ -5,12 +5,21 @@
 [Fallout](https://github.com/datastax/fallout) is a tool for running large scale remote-based distributed correctness, verification and performance tests for Apache Cassandra (TM) and Apache Pulsar (TM).
 
 Fallout uses Kubernetes and the Helm Chart to deploy the services and run the tests. The size of the cluster is limited only by the available resources.
+
+## What is Hunter
+[Hunter](https://github.com/datastax-labs/hunter) performs statistical analysis of performance test results stored in CSV files or Graphite database. 
+It finds change-points and notifies about possible performance regressions.
+
 ## What this repository is for
-This repository contains performance tests for Cassandra 
+This repository contains:
+- performance tests for Cassandra 
 databases using the [Fallout tool](https://github.com/datastax/fallout).
+- codes to create the csv file that will be fed to hunter.
```

Comment:
Can you separate out the running of fallout and fallout+hunter? Some users will only want to run the tests with fallout but not use hunter.

Reply:
Thanks, addressed at [87fcf9d](https://github.com/datastax/fallout-tests/pull/5/commits/87fcf9d6bb3b4aba5d5544739b9832fe4925feea)